src/Components/WebAssembly/JSInterop/src/WebAssemblyJSRuntime.cs

```diff
@@ -89,7 +89,7 @@ public TResult InvokeUnmarshalled<TResult>(string identifier)
         /// <param name="arg1">The second argument.</param>
         /// <param name="arg2">The third argument.</param>
         /// <returns>The result of the function invocation.</returns>
-        public TResult InvokeUnmarshalled<T0, T1, T2, TResult>(string identifier, T0 arg0, T1 arg1, T2 arg2)
+        public virtual TResult InvokeUnmarshalled<T0, T1, T2, TResult>(string identifier, T0 arg0, T1 arg1, T2 arg2)
```

Comment:
Is there a way we can avoid this virtual method? If are making this method virtual, shouldn't we make all the overloads virtual for consistency?

Reply:
What do you think about making the other overloads extension methods? They just call in to this overload with default values